src/ui/common/src/components/DataPreviewer/index.tsx

```diff
@@ -99,6 +99,8 @@ const DataPreviewer: React.FC<Props> = ({ previewData, error }) => {
               width: '100%',
               overflow: 'auto',
               overflowY: 'hidden',
+              minWidth: '400px',
+              position: 'absolute',
```

Comment:
Just out of curiosity, why do we need the absolute positioning here?

Reply:
Good question, without having position:absolute, our artifact previews were not being shown in the drawer.

I need to dig a little deeper into the stylesheet of the Drawer component, but this is what I had to do to get things to work :)